db/seeding.js

```diff
@@ -0,0 +1,88 @@
+const faker = require('faker');
+const mongoose = require('mongoose');
+
+const songIdGen = () => {
+  const min = 1;
+  const max = 100;
+  return Math.floor(Math.random() * (max - min) + min);
+};
+const commentsCreater = () => {
+  const comContainer = [];
```

Comment:
Maybe rename to "commentsContainer"? I wasn't sure what "com" was until further down.